### PR TITLE
Exclude manifest.json, robots.txt and sitemap.xml from routing matcher

### PR DESCRIPTION
**Pull request by leanderiversen**
Fix for #286

### CODE REVIEW

File: packages/open-next/src/adapters/routing/matcher.ts [hunk 1]
```diff
@@ -173,8 +173,13 @@ export function handleRedirects(
 ): InternalResult | undefined {
   if (
     NextConfig.trailingSlash &&
+    !event.headers["x-nextjs-data"] &&
     !event.rawPath.endsWith("/") &&
-    !event.headers["x-nextjs-data"]
+    ![
```
Comment (khuezy) [inline packages/open-next/src/adapters/routing/matcher.ts:178]: Do you need to apply this to the block below too?

Comment (leanderiversen) [reply packages/open-next/src/adapters/routing/matcher.ts:178]: It shouldn't need the file checks, if that's what you mean, as the next block would only be triggered if `trailingSlash` is disabled.

Comment (khuezy) [reply packages/open-next/src/adapters/routing/matcher.ts]: If there's linting issue, you can run `npm run lint:fix`

Comment (leanderiversen) [reply packages/open-next/src/adapters/routing/matcher.ts]: Sorry about that. It should be fixed now.

**Review by conico974 — CHANGES_REQUESTED** [review]
I've run some test with `next start` and it looks like next is removing trailing slash for every path that ends with an extension ( In fact they do it for everything that ends with `.SOMETHING`)
We should probably do something similar. A regex like that `/[\w-]+\.[\w]+$/g` should do the trick

**Comment by leanderiversen** [comment]
@conico974 Thank you for the feedback. This should be updated now.

**Review by conico974 — APPROVED** [review]
LGTM

**Comment by khuezy** [comment]
@leanderiversen do you need this release ASAP? If not, we'll include it in the next once the ~3 PRs are merged.

**Comment by leanderiversen** [comment]
@khuezy No, I am happy to wait. Thank you for checking!

**Pull request merged**